lib/src/dsl_util.rs

```diff
@@ -475,46 +488,53 @@ impl<P> AliasesMap<P> {
     }
 
     /// Looks up symbol alias by name. Returns identifier and definition text.
-    pub fn get_symbol(&self, name: &str) -> Option<(AliasId<'_>, &str)> {
+    pub fn get_symbol(&self, name: &str) -> Option<(AliasId<'_>, &<V as Deref>::Target)> {
```

Comment:
Maybe we can just make it return `&String` (= `&V`) instead of `&str`. For a generic container type, there wouldn't be much reason to convert ref to slice type.